### PR TITLE
Uninstall the KA Lite static.

### LINKED ISSUE #387: Uninstalling KA Lite on Windows doesn't uninstall ka-lite-static
## Summary

Uninstalling KA Lite on Windows (via the GUI approaches) seems to leave the python module behind. Not sure the full implications of this (other than not freeing up disk space, and possible issues with later reinstallations?), but figured it was good to note here, especially if it helps inform approaches to building Kolibri installers for Windows.
## Branch or installer
- Installer: Windows
- Version: 0.16.1
## Traceback or server.log snippet

The following is after uninstalling KA Lite through the GUI, on Elizabeth's computer:

```
C:\Users\Tramanh>c:\Python27\Scripts\kalite

Usage:
  kalite start [--foreground --watch] [options] [DJANGO_OPTIONS ...]
  kalite stop [options] [DJANGO_OPTIONS ...]
  kalite restart [options] [DJANGO_OPTIONS ...]
  kalite status [options]
  kalite shell [options] [DJANGO_OPTIONS ...]
  kalite test [options] [DJANGO_OPTIONS ...]
  kalite manage [options] COMMAND [DJANGO_OPTIONS ...]
  kalite diagnose [options]
  kalite -h | --help
  kalite --version

C:\Users\Tramanh>c:\Python27\Scripts\kalite start

Running 'kalite start' as daemon (system service)
Stand by while the server loads its data...
To access KA Lite from another connected computer, try the following address(es):
        http://137.110.83.48:8008/
To access KA Lite from this machine, try the following address:
        http://127.0.0.1:8008/
Going to daemon mode, logging to C:\Users\Tramanh\.kalite\server.log
```

It seems `ka-lite-static` (which was installed by the installer) isn't being uninstalled by the uninstaller:

```
C:\Users\Tramanh>c:\Python27\Scripts\pip uninstall ka-lite-static

Uninstalling ka-lite-static-0.16.1:
  c:\python27\lib\site-packages\ka_lite_static-0.16.1-py2.7.egg-info
  c:\python27\lib\site-packages\kalite
Proceed (y/n)? y
  Successfully uninstalled ka-lite-static-0.16.1
```

Even after uninstalling via pip, there still seem to be remnants under `C:\Python27\share\kalite`:

```
C:\Users\Tramanh>c:\Python27\Scripts\kalite start

c:\Python27\Scripts\kalite:129: RuntimeWarning: Could not determine Python in yo
ur path, defaulting to C:\Python27\python.exe
  warnings.warn("Could not determine Python in your path, defaulting to {0:s}".f
ormat(python_executable), RuntimeWarning)
Traceback (most recent call last):
  File "C:\Python27\share\kalite\kalitectl.py", line 115, in <module>
    import kalite
ImportError: No module named kalite
```
## How to reproduce
1. Install KA Lite 0.16.1
2. Uninstall KA Lite
3. Try the commands shown in the tracebacks above.

### PR DESCRIPTION
**Pull request by mrpau-richard**
## Summary 
Remove KA lite static during KA Lite application uninstall. 

fixes #387

### CODE REVIEW

**Comment by mrpau-richard** [comment]
Hi @radinamatic can you test the fix using this windows [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMNnBxVmdZOV9hcVU)

**Comment by radinamatic** [comment]
Tested on Win 7

virtualbox_ie11 - win7_10_01_2017_21_33_33

File: windows/installer-source/KaliteSetupScript.iss [hunk 1]
```diff
@@ -333,7 +336,7 @@ begin
              HandlePythonSetup();
              exit;
         end
-    else
+    else
```
Comment (cpauya) [inline windows/installer-source/KaliteSetupScript.iss]: Can you remove the extra `^M` characters in this PR if they are not needed?

**Comment by mrpau-richard** [comment]
@cpauya done removing the extra ^M characters at `KaliteSetupScript.iss`. But I was not able to remove the extra character at `uninstall-ka-lite-static.bat`.

**Comment by cpauya** [comment]
Sorry, that was me who merged this.

**Pull request merged**